src/CodepageTranslator.cc

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2020 Peter Kessen
+ *
+ * This file is part of dotprint.
+ *
+ * dotprint is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * dotprint is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with dotprint. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "CodepageTranslator.h"
+#include <fstream>
+#include <iostream>
+#include <iomanip>
+
+CodepageTranslator::CodepageTranslator()
```

Comment:
So do you want me to delete it?
IMO a class always has a constructor so i write it down in the code and it can easily be added some code to that constructor  if it is necessary.

Reply:
I don't think there is any benefit in writing these out.

If you wanted to write out all the member functions that a class has, then you'd also have to write the copy and move ctors and copy and move assignment operator.

Reply:
I removed them for now. If we need it we can add them again

Reply:
Thank.